x-pack/test/functional/apps/maps/discover.js

```diff
@@ -35,8 +35,8 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.maps.waitForLayersToLoad();
       const doesLayerExist = await PageObjects.maps.doesLayerExist('geo_shapes*');
       expect(doesLayerExist).to.equal(true);
-      const hits = await PageObjects.maps.getHits();
-      expect(hits).to.equal('4');
+      const tooltipText = await PageObjects.maps.getLayerTocTooltipMsg('geo_shapes*');
+      expect(tooltipText).to.equal('geo_shapes*\nFound 8 documents.');
```

Comment:
Why did 4 change to 8? Some how, vector tiles are double counting shapes. Will need to investigate issue but this is an existing bug and not an issue with this PR.

Reply:
I suspect it has something to do with shapes crossing tile boundaries  and getting double counted.